src/flow.ts

```diff
@@ -228,8 +221,24 @@ export class Flow {
    * @returns an array of Font objects corresponding to the provided `fontNames`.
    */
   static setMusicFont(...fontNames: string[]): Font[] {
+    // HACK during review to introduce the correct stacks step by step
+    switch (fontNames[0]) {
+      case 'Bravura':
+        CommonMetrics.fontFamily = 'Bravura,BravuraText';
+        break;
+      case 'Gonville':
+        CommonMetrics.fontFamily = 'GonvilleSmufl,Bravura,Roboto Slab';
+        break;
+      case 'Leland':
+        CommonMetrics.fontFamily = 'Leland,Bravura,LelandText';
```

Comment:
Same here. Drop LelandText from the font stack.

Reply:
done